src/FormModelInterface.php

```diff
@@ -230,4 +230,11 @@ public function validate(): bool;
      * @param mixed $value value
      */
     public function setAttribute(string $name, $value): void;
+
+    /**
+     * This method allows to know if the validation was executed or not in the model.
+     *
+     * @return bool
+     */
+    public function isValidate(): bool;
```

Comment:
```suggestion
    public function isValidated(): bool;
```
Maybe this is better?

Reply:
done.